test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -898,14 +895,31 @@ UCS_TEST_P(test_uct_cm_sockaddr, listener_query)
 UCS_TEST_P(test_uct_cm_sockaddr, cm_open_listen_close)
 {
     skip_tcp_sockcm();  /* disconnect for tcp_sockcm isn't implemented yet */
-    listen_and_connect();
+    basic_listen_connect_disconnect();
+}
 
-    wait_for_bits(&m_state, TEST_STATE_SERVER_CONNECTED |
-                            TEST_STATE_CLIENT_CONNECTED);
-    EXPECT_TRUE(ucs_test_all_flags(m_state, (TEST_STATE_SERVER_CONNECTED |
-                                             TEST_STATE_CLIENT_CONNECTED)));
+UCS_TEST_P(test_uct_cm_sockaddr, cm_open_listen_close_large_priv_data)
+{
+    skip_tcp_sockcm();
+    m_entities.clear();
 
-    cm_disconnect(m_client);
+    /* Set the values for max send/recv socket buffers (for tcp_sockcm) to
+     * small enough values to have the send/recv of a large data buffer in
```

Comment:
what happens on rdmacm?

Reply:
the config_modify won't affect rdmacm and it will run a short test of listen, connect and disconnect.

Reply:
ok